content/docs/command-reference/add.md

```diff
@@ -144,9 +143,12 @@ directory symlinks.
 - `--external` - allow `targets` that are outside of the DVC repository. See
   [Managing External Data](/doc/user-guide/managing-external-data).
 
-- `--glob` - allows adding files and directories that match the specified
-  pattern as specified by `target`. Shell-style wildcards are supported: `*`,
-  `?`, `[seq]`, `[!seq]`, and `**`.
+  > Note that this option implies `--no-commit`, as external outputs are never
+  > pushed or pulled from/to remote storage. See link above for more details.
+
+- `--glob` - allows adding files and directories that match the
```

Comment:
also, looks like `--glob` is out of order with the other options?

Reply:
I guess it can go before --external? I'm not sure which one we expect people to use more. I moved it up one place for now.

Reply:
it should be in the same order as DVC CLI has it in, right?

Reply:
Not necessarily. We use an arbitrary order here. The core team tends to always append options at the end. There's no consistency so far 🙁.

There's a discussion about this in iterative/dvc/issues/3642.

Reply:
kk